docs/sphinx/source/conf.py

```diff
@@ -53,7 +53,8 @@
 # directories to ignore when looking for source files.
 # This pattern also affects html_static_path and html_extra_path.
 
-exclude_patterns = []
+exclude_patterns = ['changelog/*']
```

Comment:
This just prevents sphinx from emitting `v2.0.0.rst: WARNING: document isn't included in any toctree`